src/main/scala/akka/persistence/cassandra/query/SequenceNumbers.scala

```diff
@@ -0,0 +1,43 @@
+package akka.persistence.cassandra.query
+
+private[query] object SequenceNumbers {
+  val empty = SequenceNumbers(Map.empty, Map.empty)
+
+  sealed trait Answer
+  case object Yes extends Answer
+  case object Before extends Answer
+  case object After extends Answer
+  case object PossiblyFirst extends Answer
+}
+
+private[query] final case class SequenceNumbers(
+  intNumbers: Map[String, Int], longNumbers: Map[String, Long]) {
```

Comment:
Saving memory. Long sequence numbers are only a theoretical construct, saving more than `Int.MaxValue` event for one persistence would be insane, especially replaying that stream.